muhasin/index.js

```diff
@@ -0,0 +1,49 @@
+// 1. use NESTED LOOP in each instance
+
+let pattern = "";
+for (let i = 1; i <= 4; i++) {
+    pattern += "* ";
+    console.log(pattern);
+
+}
+
+// 2. Write a program which prints the elements of the following array:
+let arr = [
+    [1, 2, 1, 24],
+    [8, 11, 9, 4],
+    [7, 0, 7, 27],
+    [7, 4, 28, 14],
+    [3, 10, 26, 7]
+];
+
+for (let i = 0; i < arr.length; i++) {
```

Comment:
task 2 ==> It prints just row 0, row 1 , row 2,row 3 and row 4. The result should be like this. 
row 0 
 1 
 2 
 1
 24
row 1
8
11
9
2 and so on...